### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by vladyslavchichkan**

### CODE REVIEW

File: cinema/views.py [hunk 1]
```diff
@@ -21,7 +24,7 @@
     MovieSessionDetailSerializer,
     MovieListSerializer,
     OrderSerializer,
-    OrderListSerializer,
+    OrderListSerializer, MovieImageSerializer,
```
Comment (MNDonut) [inline cinema/views.py]: we don't add 2+ imports on the same line when use ()

File: cinema/views.py [hunk 2]
```diff
@@ -97,12 +100,27 @@ def get_queryset(self):
...
 
+    @action(
+        methods=["POST"],
+        detail=True,
+        url_path="upload-image"
+    )
+    def upload_image(self, request, pk=None):
+        movie = get_object_or_404(Movie, pk=pk)
+        serializer = self.get_serializer(movie, data=request.data)
+        print(request.data)
```
Comment (MNDonut) [inline cinema/views.py]: you can remove this print ;)

File: cinema/views.py [hunk 3]
```diff
@@ -97,12 +100,27 @@ def get_queryset(self):
...
+        if serializer.is_valid():
+            serializer.save()
+            return Response(serializer.data, status=status.HTTP_200_OK)
+        return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
```
Comment (MNDonut) [inline cinema/views.py]: you can simplify it calling serializer.is_valid(raise_exception=True) separately, thus there won't be a need to add if statement

File: user/serializers.py [hunk 4]
```diff
@@ -22,3 +31,41 @@ def update(self, instance, validated_data):
             user.save()
 
         return user
+
+
+class AuthTokenSerializer(serializers.Serializer):
+    email = serializers.CharField(
```
Comment (MNDonut) [inline user/serializers.py]: you can use EmailField here

**Review by MNDonut — CHANGES_REQUESTED** [review]

**Review by Arsen-hrynevych — APPROVED** [review]
Thanks

**Pull request closed without merging**